packages/flutter_tools/lib/src/macos/cocoapods.dart

```diff
@@ -371,7 +371,7 @@ class CocoaPods {
       ).send();
       _logger.printError(
         'Error: To set up CocoaPods for ARM macOS, run:\n'
-        '  arch -x86_64 sudo gem install ffi\n',
+        '  sudo gem uninstall ffi && sudo gem install ffi -- --enable-libffi-alloc\n',
```

Comment:
Does the user need to have a specific gem version to have this flag? Or is it old enough we don't have to worry?

Reply:
If they run this command they will get the newest version.

Reply:
nice